typescript/phoenix-sdk/src/utils/token.ts

```diff
@@ -0,0 +1,35 @@
+import { Connection, PublicKey, TransactionInstruction } from "@solana/web3.js";
+import {
+  createAssociatedTokenAccountInstruction,
+  getAssociatedTokenAddress,
+} from "@solana/spl-token";
+
+// Check if token account exists, if not, return create token account instructions
```

Comment:
change first token account to "associated token account", also period instead of first comma.